### PR TITLE
fixed transmission loss plotting

### PR DESCRIPTION
**Pull request by Vanillaholic**
Sir, the attribute of tloss maybe not correspond after computing the tloss, which leads to wrong x-axis when plotting , variable environment is needed.This is the submitted PR.

### CODE REVIEW

File: arlpy/uwapm.py [hunk 1]
```diff
@@ -524,18 +524,18 @@ def plot_transmission_loss(tloss, env=None, **kwargs):
...
     """
-    xr = (min(tloss.columns), max(tloss.columns))
+    if env is None:
+        raise ValueError("Environment is required")
+    xr = (env['rx_range'].min(), env['rx_range'].max())
     yr = (-max(tloss.index), -min(tloss.index))
     xlabel = 'Range (m)'
     if xr[1]-xr[0] > 10000:
-        xr = (min(tloss.columns)/1000, max(tloss.columns)/1000)
+        xr = (env['rx_range'].min()/1000, env['rx_range'].max()/1000)
```
Comment (mchitre) [inline arlpy/uwapm.py:535]: Is the problem that `tloss.columns` is not correctly populated? If so, perhaps that's the place to fix this?

**Review by mchitre — CHANGES_REQUESTED** [review]

File: arlpy/uwapm.py (hunk 1 above)
Comment (Vanillaholic) [reply arlpy/uwapm.py:535]: yes，'tloss.columns' is incorrect , but it was calculated by Acoustic Toolbox in fortran. Could only fix it  in original package

Comment (mchitre) [reply arlpy/uwapm.py:535]: Fair. I suggest that we still make `env` optional (recommended), but pick up `rx_range` from it if it is specified. Works?

File: arlpy/uwapm.py [hunk 2]
```diff
@@ -524,18 +524,18 @@ def plot_transmission_loss(tloss, env=None, **kwargs):
             max_angle=45
         )
     >>> tloss = pm.compute_transmission_loss(env)
-    >>> pm.plot_transmission_loss(tloss, width=1000)
+    >>> pm.plot_transmission_loss(tloss,env, width=1000)
```
[the remark this answers is not in the document]
Comment (mchitre) [reply arlpy/uwapm.py:527]: This is fine if we are making it recommended.
